common-app-lib/src/cmr/common_app/app_events.clj

```diff
@@ -54,7 +54,7 @@
   "Add a shutdown hook to call the CMR app stop command found in most if not all
    CMR microservices. Caller must supply a function that calls the application's
    stop function such as: #(cmr.search.system/stop system)"
-  [app-name stop-thread]
+  [_app-name stop-thread]
```

Comment:
Get rid of _app-name in this function and then remove (:instance-name system) from the following files:
ingest-app/src/cmr/ingest/runner.clj
search-app/src/cmr/search/runner.clj 

from the stop-on-exit-hook function

Reply:
I think I added those so that other apps could print out the name of themselves as they were being killed.

Reply:
Good point Thomas, we missed the purpose/the supporter here. Reverting change